### DISCUSSION TITLE
required popup turn off

### DISCUSSION THREAD

**Discussion by enessvg** [filamentphp/filament#14271] [Help] [answered]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

Hi friends, I want to turn off the thing that appears here when I put required as shown in the 1st picture. please help
<img width="236" alt="Ekran Resmi 2024-09-17 09 19 00" src="https://github.com/user-attachments/assets/542573af-28fe-4d03-b43c-e9844950c9fe">

I want verification like this one below
<img width="294" alt="Ekran Resmi 2024-09-17 09 21 33" src="https://github.com/user-attachments/assets/b7873742-286b-41f0-bc68-3c904d85d05a">

**Comment by leandrocfe** [comment]
Remove required() and use `->rules(['required'])`

**Reply by enessvg** [reply]
but this time there is no red * above the input

**Reply by leandrocfe** [reply, marked as answer]
use `->markAsRequired()`

**Reply by enessvg** [reply]
Thank you very much, it helped me. Where can I find more information like this?


okeyy, I've got it: https://filamentphp.com/docs/3.x/forms/fields/getting-started#marking-a-field-as-required
